src/thor/bidirectional_astar.cc

```diff
@@ -322,6 +323,9 @@ inline bool BidirectionalAStar::ExpandForwardInner(GraphReader& graphreader,
                                    dist, mode_, transition_cost,
                                    (pred.not_thru_pruning() || !meta.edge->not_thru()),
                                    (pred.closure_pruning() || !costing_->IsClosed(meta.edge, tile)),
+                                   static_cast<bool>(flow_sources &
+                                                     (kFreeFlowMask | kConstrainedFlowMask |
+                                                      kPredictedFlowMask | kCurrentFlowMask)),
                                    restriction_idx);
```

Comment:
Saving the flow information to the edgelabel. If we have one of the flow sources: kFreeFlowMask, kConstrainedFlowMask, kPredictedFlowMask, or kCurrentFlowMask then we have flow data and set the bit to true.

Reply:
```c++
kFreeFlowMask | kConstrainedFlowMask | kPredictedFlowMask | kCurrentFlowMask
```
- this is used several times, probably would be better to change it with a new constants ? `kFlowMask`

Reply:
Yes, plus one on this

Reply:
we have that already: `kDefaultFlowMask`

Reply:
@genadz @PureW @kevinkreiser thank you i missed that one